hw/top_darjeeling/lint/top_darjeeling_lint_cfgs.hjson

```diff
@@ -0,0 +1,324 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+
+  // This is the primary cfg hjson for RTL linting. It imports ALL individual lint
+  // cfgs of the IPs and the full chip used in top_darjeeling. This enables to run
+  // them all as a regression in one shot.
+  name: top_darjeeling_batch
+
+  flow: lint
+
+  import_cfgs:      [// common server configuration for results upload
+                     "{proj_root}/hw/data/common_project_cfg.hjson"
+                     // tool-specific configuration
+                     "{proj_root}/hw/lint/tools/dvsim/{tool}.hjson"]
+
+  // Different dashboard output path for each tool
+  rel_path: "hw/top_darjeeling/lint/{tool}/summary"
+
+  // Severities to be printed in the summary report
+  report_severities: ["warning", "error"]
+
+  use_cfgs: [
+            // {    name: mbx
+            //       fusesoc_core: lowrisc:ip:mbx
+            //       import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
+            //       rel_path: "hw/ip/mbx/lint/{tool}"
+            //  },
+            {    name: dma
+                 fusesoc_core: lowrisc:ip:dma
+                 import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
+                 rel_path: "hw/ip/dma/lint/{tool}"
+            },
+            {     name: aes
+                  fusesoc_core: lowrisc:ip:aes
+                  import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
+                  rel_path: "hw/ip/aes/lint/{tool}"
+             },
+             {    name: alert_handler
```

Comment:
I don't think this can land until the corresponding autogenerated IP is in, can it? Maybe this item needs commenting? Similarly for quite a few other blocks below.